src/routes/routes.tsx

```diff
@@ -83,6 +84,7 @@ export const AppRoutes = () => {
       <Route path={paths.circles} element={<CirclesPage />} />
       <Route path={paths.createCircle} element={<CreateCirclePage />} />
       <Route path={paths.developers} element={<DevPortalPage />} />
+      <Route path={paths.discord(':snowflake')} element={<DiscordPage />} />
```

Comment:
Is this just for prototyping? I think this flow makes sense, but we're going to receive the token via [discord's OAuth flow](https://discordjs.guide/oauth2/#authorization-code-grant-flow), right?

Reply:
Correct, implemented an MVP of the discord's authorization code grant flow with https://github.com/coordinape/coordinape/pull/1625/commits/0ed646842159f67e192d622b816c5fdbcbfb837c

We will get the following JSON as a response:
```json
{
	"access_token": "an access token",
	"token_type": "Bearer",
	"expires_in": 604800,
	"refresh_token": "a refresh token",
	"scope": "identify"
}
```
Was the idea to add that data to the discord's `users` table? We will need to keep the refresh token in hand, so when the original access token expires we will need to request a new one [like so](https://discord.com/developers/docs/topics/oauth2#authorization-code-grant-refresh-token-exchange-example).